commons/src/test/java/org/openmrs/eip/camel/route/GetConceptByMappingFromOpenmrsRouteTest.java

```diff
@@ -21,21 +21,23 @@
 import org.apache.camel.model.ProcessDefinition;
 import org.apache.camel.model.ToDynamicDefinition;
 import org.apache.camel.support.DefaultExchange;
-import org.junit.Before;
-import org.junit.Test;
+import org.apache.camel.test.spring.junit5.DisableJmx;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.openmrs.eip.AppContext;
 import org.openmrs.eip.BaseCamelTest;
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.TestPropertySource;
 
+@DisableJmx
 @TestPropertySource(properties = "spring.liquibase.enabled=false")
-@TestPropertySource(properties = "camel.springboot.xml-routes=classpath*:camel/" + ROUTE_ID_GET_CONCEPT_BY_MAPPING + ".xml")
 @TestPropertySource(properties = "logging.level." + ROUTE_ID_GET_CONCEPT_BY_MAPPING + "=DEBUG")
 @TestPropertySource(properties = "logging.level.org.apache.camel.reifier.RouteReifier=WARN")
-@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
 @TestPropertySource(properties = "openmrs.baseUrl=" + GetConceptByMappingFromOpenmrsRouteTest.OPENMRS_URL)
 @TestPropertySource(properties = "openmrs.username=" + GetConceptByMappingFromOpenmrsRouteTest.OPENMRS_USER)
 @TestPropertySource(properties = "openmrs.password=" + GetConceptByMappingFromOpenmrsRouteTest.OPENMRS_PASS)
+@TestPropertySource(properties = "camel.springboot.routes-include-pattern=classpath:camel/get-concept-by-mapping-from-openmrs.xml")
```

Comment:
Can we remain consistent and use the respective variable `ROUTE_ID_GET_CONCEPT_BY_MAPPING`...